pkg/scheduler/factory/factory.go

```diff
@@ -653,17 +653,20 @@ func NewPodInformer(client clientset.Interface, resyncPeriod time.Duration) core
 func MakeDefaultErrorFunc(client clientset.Interface, podQueue internalqueue.SchedulingQueue, schedulerCache internalcache.Cache, stopEverything <-chan struct{}) func(pod *v1.Pod, err error) {
 	return func(pod *v1.Pod, err error) {
 		if err == core.ErrNoNodesAvailable {
-			klog.V(4).Infof("Unable to schedule %v/%v: no nodes are registered to the cluster; waiting", pod.Namespace, pod.Name)
+			klog.V(2).Infof("Unable to schedule %v/%v: no nodes are registered to the cluster; waiting", pod.Namespace, pod.Name)
 		} else {
 			if _, ok := err.(*core.FitError); ok {
-				klog.V(4).Infof("Unable to schedule %v/%v: no fit: %v; waiting", pod.Namespace, pod.Name, err)
+				klog.V(2).Infof("Unable to schedule %v/%v: no fit: %v; waiting", pod.Namespace, pod.Name, err)
 			} else if errors.IsNotFound(err) {
+				klog.V(2).Infof("Unable to schedule %v/%v: possibly due to node not found: %v; waiting", pod.Namespace, pod.Name, err)
 				if errStatus, ok := err.(errors.APIStatus); ok && errStatus.Status().Details.Kind == "node" {
 					nodeName := errStatus.Status().Details.Name
+					klog.V(2).Infof("Unable to schedule %v/%v: node %v not found; going to try to get the node again.", pod.Namespace, pod.Name, nodeName)
 					// when node is not found, We do not remove the node right away. Trying again to get
 					// the node and if the node is still not found, then remove it from the scheduler cache.
 					_, err := client.CoreV1().Nodes().Get(nodeName, metav1.GetOptions{})
 					if err != nil && errors.IsNotFound(err) {
+						klog.V(2).Infof("Unable to schedule %v/%v: removing node %v from cache since it's not found after retry.", pod.Namespace, pod.Name, nodeName)
```

Comment:
I don't think this or the one at line 664 are useful at v=2. I think we may not need them at all.

Reply:
The one at L664 may not be very useful. But isn't it useful to know when a node is removed from the cache? This can inform us why a node is not picked for example.

Reply:
we already have that log: https://github.com/kubernetes/kubernetes/pull/80811/files#diff-25c437f9d2edd767aa5384a720a9eea4R113

Reply:
Yes we have the log for RemoveNode in cache. I was thinking that cache.RemoveNode can be called in different places where the reason to remove a node may be different. So it's still useful to log it here why we need to remove the node.

But I don't feel strongly about this.

Reply:
What I think we are trying to do in this PR is increase the visibility of a couple of log lines that help us track the state of  the scheduler's view of pods/nodes. I don't think one and the one in line 664 are necessary to be honest, we can easily infer that if we see a node remove log after a log from line 661

Reply:
Removed.